sdks/python/apache_beam/testing/load_tests/load_test.py

```diff
@@ -17,19 +17,50 @@
 from __future__ import absolute_import
 
 import json
-import logging
-import unittest
 
 from apache_beam.metrics import MetricsFilter
 from apache_beam.testing.load_tests.load_test_metrics_utils import MetricsReader
 from apache_beam.testing.test_pipeline import TestPipeline
 
 
-class LoadTest(unittest.TestCase):
-  def parseTestPipelineOptions(self, options=None):
+class LoadTest(object):
+  def __init__(self):
+    self.pipeline = TestPipeline(is_integration_test=True)
```

Comment:
If we proceed with this change, do we still need to use TestPipeline as opposed to regular pipeline?

Reply:
It's just a matter of passing pipeline options in a different way (without `--test-pipeline-options` command line argument). Do you think it might be beneficial to use regular pipeline instead of TestPipeline?

Reply:
I'd let @markflyhigh  comment on which way is more idiomatic/preferable going forward for adding new tests. Without test-pipeline-options we will be declaring options explicitly, it may be easier to see which flags need to be passed.

Reply:
I spoke with @udim - if we migrate these tests to use pytest later, sticking with `--test-pipeline-options` may ease this migration. we haven't fully paved the path to switch to pytest, but @udim may have some in-progress PRs in case you are curious to give pytest a try for loadtest purposes.

Reply:
In this case, I'd rather stick with `--test-pipeline-options` for now. Thanks for the link.